### PR TITLE
Ihadss removal code update

### PR DESCRIPTION
**Pull request by Ollieollieolllie**

### CODE REVIEW

**Review by mattysmith22 — APPROVED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_controller.sqf [hunk 1]
```diff
@@ -20,16 +20,16 @@ Author:
 ---------------------------------------------------------------------------- */
 params ["_heli"];
 
-if (vehicle player != _heli && !(vehicle player isKindOf "fza_ah64base") || !(alive _heli) && !(vehicle player isKindOf "fza_ah64base") || !(alive player) || !(isNull curatorCamera)) exitWith {
```
[the remark this answers is not in the document]
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_controller.sqf:23]: already done this, your reading the old code

File: @AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_controller.sqf [hunk 2]
```diff
@@ -20,16 +20,16 @@ Author:
...
-    1 cuttext["", "PLAIN"];
-    2 cuttext["", "PLAIN"];
-    3 cuttext["", "PLAIN"];
-    4 cuttext["", "PLAIN"];
+if (!(vehicle player isKindOf "fza_ah64base") || !(alive _heli) || !(alive player) || !(isNull curatorCamera)) exitWith {
+    ((uiNameSpace getVariable "fza_ah64_nvsoverlay") displayCtrl 120) ctrlSetText "";
```
[the remark this answers is not in the document]
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_controller.sqf:24]: its leaves a black trasparent box on screen without it

File: @AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_controller.sqf [hunk 3]
```diff
@@ -20,7 +20,7 @@ Author:
 ---------------------------------------------------------------------------- */
 params ["_heli"];
 
-if (!(vehicle player isKindOf "fza_ah64base") || !(alive _heli) || !(alive player) || !(isNull curatorCamera)) exitWith {
+if (!(vehicle player isKindOf "fza_ah64base") || !(alive _heli) || !(alive player) || !(isNull curatorCamera) || is3DEN || !(cameraView in ["INTERNAL", "GUNNER"]) || (isRemoteControlling player && !(getConnectedUAV player isKindOf "fza_ah64base"))) exitWith {
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_controller.sqf:23]: This is starting to get a bit long for a single line, but not sure the best way to simplify

**Review by mattysmith22 — APPROVED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_controller.sqf (hunk 3 above)
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_controller.sqf:23]: readability 5/10
functionality 10/10

**Pull request merged**